forms/core/Ajax.php

```diff
@@ -235,7 +236,6 @@ public static function ajaxFormHandler()
 		);
 
 		$CampaignMonitor = Application::$CampaignMonitor;
-		$x = $CampaignMonitor->get_clients();

```

Comment:
Could you ensure that this overall method still work as intended. Although the result of get_clients() may not be used, that code may set up authorization for the API, which might be needed on subsequent call such as add_subscriber.

Reply:
get_clients() doesn't set anything if the credentials are empty so I have no idea why they call this method here.